From 0a392baf1874964651115d9f77b0daa6851d1daa Mon Sep 17 00:00:00 2001
From: Jonathan Liu <net147@gmail.com>
Date: Tue, 1 Mar 2016 14:28:01 +1100
Subject: [PATCH] Revert "mke2fs: enable the metadata_csum and 64bit features
 by default"

This reverts commit cd27af3ecb83e8fd1e3eaa14994284a1818c7c15 as we
don't want to enable features by default that are not supported by
the latest stable e2fsprogs release.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Jonathan Liu <net147@gmail.com>
---
 misc/mke2fs.conf.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index a049d61..106ee80 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,7 +11,8 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
+		features = has_journal,extent,huge_file,flex_bg,uninit_bg,dir_nlink,extra_isize
+		auto_64-bit_support = 1
 		inode_size = 256
 	}
 	ext4dev = {
-- 
2.7.2

